tests/Files.InteractionTests/Tests/FolderTests.cs

```diff
@@ -12,34 +12,7 @@ public class FolderTests
 
 		[TestCleanup]
 		public void Cleanup()
-		{
-			var action = new Actions(SessionManager.Session);
-
-			// Select the "Renamed Folder" folder and click the delete button
-			TestHelper.InvokeButtonByName("Renamed Folder");
-			TestHelper.InvokeButtonById("Delete");
-
-			// Wait for prompt to show
-			Thread.Sleep(1000);
-
-			// Press the enter key
-			action.SendKeys(Keys.Enter).Perform();
-
-			// Select the "Renamed Folder - Copy"  folder and click the delete button
-			TestHelper.InvokeButtonByName("Renamed Folder - Copy");
-			TestHelper.InvokeButtonById("Delete");
-
-			// Wait for prompt to show
-			Thread.Sleep(1000);
-
-			// Press the enter key
-			action = new Actions(SessionManager.Session);
-			action.SendKeys(Keys.Enter).Perform();
-
-			// Wait for items to be deleted
-			Thread.Sleep(1000);
-
-
+		{			
```

Comment:
Should we still call the folder test in the cleanup to be sure?

Reply:
What do you mean?

Reply:
If the test crashes for some reason, we would still leave the test folder on disk right? Calling the delete folder test could help us clean that up. Alternatively, if we know for sure where the test folder is created, we could also delete it using C# file API.